handle complex datatype

nikopj opened the pull request:
Addresses #56. Tests added for ComplexF64.

Feedback appreciated.

nikopj committed:
handle complex datatype

avik-pal commented:
Any idea what the test failures are? https://buildkite.com/julialang/nccl-dot-jl/builds/36#0190cca1-710b-4b33-96b5-39d1fc64a262/303-1126

nikopj commented:
> Any idea what the test failures are? https://buildkite.com/julialang/nccl-dot-jl/builds/36#0190cca1-710b-4b33-96b5-39d1fc64a262/303-1126

This looks unrelated to the pull request to me... did a configuration for the tests change since the last successful PR?

findmyway commented:
The failure is the same with https://github.com/JuliaGPU/NCCL.jl/issues/58 and should be fixed  with https://github.com/JuliaGPU/CUDA.jl/pull/2457

nikopj commented:
Can we retrigger the build test now that https://github.com/JuliaGPU/CUDA.jl/pull/2457 is merged?

avik-pal commented:
do a force push to your branch, it will retrigger the build. I don't have permissions

nikopj committed:
added whitespace to force push

avik-pal commented:
That patch hasn't been released yet, we will have to wait till it is released

avik-pal commented:
Maybe we should also bump CUDA version to that patch once it is released

nikopj committed:
complex data handling

nikopj commented:
Trying to get this issue resolved again now that the CUDA patch from earlier is released. Not sure if this build failure is my fault... any ideas @avik-pal @simonbyrne

avik-pal commented:
I think we need to bump the minimum julia versions in ci

simonbyrne commented:
That's fine by me.
